resources/prometheus/federation-config.yaml

```diff
@@ -1,4 +1,171 @@
 match[]:
```

Comment:
Can you mark this file as generated? Maybe also move it to a `generated` folder.

Reply:
I have marked the file as generated. There are also two other generated files in the same folder. If we want to move them to the `generated` subfolder, I would do it in a separate PR.